### DISCUSSION TITLE
Confused on how the config.json file works

### DISCUSSION THREAD

**Discussion by Christophvh** [soketi/pws#369] [Q&A]
Hi, Thanks for the great project!

I got soketi working locally and on a server with de default configuration. But i would like to set app_id/app_secret etc.

https://docs.soketi.app/getting-started/environment-variables
the above page in the docs gives this example:
```
{
    "appManager.array.apps.0.id": "some-id",
    "appManager.array.apps.0.key": "some-key",
    "appManager.array.apps.0.secret": "some-secret",
    "appManager.array.apps.0.webhooks": [{
        "url": "https://...",
        "event_types": ["channel_occupied"],
    }],
    "debug": true,
    "port": 6002
}
```

but I'm a bit confused on what the '0' refers too, is this just the syntax for the first app? or is the app-id?
I made following config file and ran it with a daemon. But the connection doesn't  work. Am i missing something? is there any example default config that is suitable for a multi-site setup on a server? Or an example which is good for a production environment?

{
    "appManager.array.apps.0.id": "test-development",
    "appManager.array.apps.0.key": "test-development-key",
    "appManager.array.apps.0.secret": "blabla",
    "appManager.array.apps.1.id": "test-staging",
    "appManager.array.apps.1.key": "test-staging-key",
    "appManager.array.apps.1.secret": "blablaqsdfqsdfs",
}

**Comment by rennokki** [comment]
The file is using dot notation: https://github.com/wadackel/dot-wild

`0` refers to the first item of the array. Your connection issues can be tracked in the Developer Tools feature from your browser and see what WS connections are issued.

**Reply by Christophvh** [reply]
No console errors, but websocket functionality stopped working with new id/key/secret values. 

When I revert them back to the app-id/app-key/app-secret defaults it works instantly.

What i tried:
- Redeployed app, so caching isn't an issue.
- Restarted daemon
